.github/workflows/windows.yml

```
@@ -139,3 +139,75 @@ jobs:
   #        name: pr_number
   #        path: pr_number.txt
   #        retention-days: 1
+
+  # Build libamrex and all tutorials with CUDA
+  tutorials-cuda:
+    name: MSVC C++17 w/ CUDA@12.4 w/o Fortran w/o MPI
+    runs-on: windows-2022
+    steps:
+    - uses: actions/checkout@v4
+    - name: Download CUDA
+      run: |
+        if (Test-Path "C:\Program Files\NVIDIA GPU Computing Toolkit\") { exit }
+        New-item -ItemType directory -Name cuda
+        Set-Location -Path cuda -PassThru
+        $cuda_version_short = '12.4'
+        Invoke-WebRequest -Uri 'https://developer.download.nvidia.com/compute/cuda/12.4.0/network_installers/cuda_12.4.0_windows_network.exe' -OutFile 'cuda_install.exe'
+
+        $CudaFeatures = " nvcc_${cuda_version_short} " + `
+  " cublas_${cuda_version_short} " + `
+  " cublas_dev_${cuda_version_short} " + `
+  " cuda_profiler_api_${cuda_version_short} " + `
+  " cudart_${cuda_version_short} " + `
+  " cufft_${cuda_version_short} " + `
+  " cufft_dev_${cuda_version_short} " + `
+  " cuobjdump_${cuda_version_short} " + `
+  " cupti_${cuda_version_short} " + `
+  " curand_${cuda_version_short} " + `
+  " curand_dev_${cuda_version_short} " + `
+  " cusolver_${cuda_version_short} " + `
+  " cusolver_dev_${cuda_version_short} " + `
+  " cusparse_${cuda_version_short} " + `
+  " cusparse_dev_${cuda_version_short} " + `
+  " cuxxfilt_${cuda_version_short} " + `
+  " npp_${cuda_version_short} " + `
+  " npp_dev_${cuda_version_short} " + `
+  " nsight_vse_${cuda_version_short} " + `
+  " nvdisasm_${cuda_version_short} " + `
+  " nvjitlink_${cuda_version_short} " + `
+  " nvjpeg_${cuda_version_short} " + `
+  " nvjpeg_dev_${cuda_version_short} " + `
+  " nvml_dev_${cuda_version_short} " + `
+  " nvprof_${cuda_version_short} " + `
+  " nvprune_${cuda_version_short} " + `
+  " nvrtc_${cuda_version_short} " + `
+  " nvrtc_dev_${cuda_version_short} " + `
+  " nvtx_${cuda_version_short} " + `
+  " sanitizer_${cuda_version_short} " + `
+  " thrust_${cuda_version_short} "
+
+        $SilentFlag = '-s '
+        Start-Process -FilePath '.\cuda_install.exe' -ArgumentList @($SilentFlag + $CudaFeatures) -Wait -NoNewWindow
+        Remove-Item cuda_install.exe
+    - name: Build & Install
+      shell: cmd
+      run: |
+        call "C:\Program Files (x86)\Microsoft Visual Studio\2022\Enterprise\vc\Auxiliary\build\vcvarsall.bat" x64
+        SET "CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.4"
```

For completeness we could also set `CUDA_HOME`:
```suggestion
        SET "CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.4"
        SET "CUDA_HOME=%CUDA_PATH%"
```